lib/models/NumericalDataSeries.js

```diff
@@ -3,19 +3,33 @@ import DataSeries from './DataSeries';
 class NumericalDataSeries extends DataSeries {
     constructor(data, options) {
         /*
-        *  Data is in the form of:
-        *     [{x_value: 1, y_value: 2}, {x_value: 3, y_value: 4}, etc]
-        */
+         *  Data is in the form of:
+         *     [{
+         *         x_value : 1,
+         *         x_min   : 0,
+         *         x_max   : 2,
+         *         y_value : 2,
+         *         y_min   : 1,
+         *         y_max   : 3
+         *     }, {
+         *         x_value : 3,
+         *         x_min   : 1,
+         *         x_max   : 4,
+         *         y_value : 4,
+         *         y_min   : 3,
+         *         y_max   : 5
+         *     }, etc]
+         */
```

Comment:
Did you mention that this could accept a few different flavors of data? If so, it might be helpful to add examples of each here.